### DISCUSSION TITLE
can we overide the 'ESP32  MP_TASK_STACK_SIZE'?

### DISCUSSION THREAD

**Discussion by dotnfc** [micropython/micropython#12408] [ESP32] [answered]
When developing an extmodule, we may need to increase the MP_TASK_STACK_SIZE larger than 16KB of the port/esp32. 

Currently, i just modify the ports/esp32/main.c file directly. 

Is it possible to specify the value of MP_TASK_STACK_SIZE in the make command line or in a board configuration file to overide it?

**Comment by jimmo** [comment, marked as answer]
> Is it possible to specify the value of MP_TASK_STACK_SIZE in the make command line or in a board configuration file to overide it?

Not in the current code. If you wanted to send a PR to make it overridable from a board definition then that would be fine.

FWIW, I think the macro should be renamed to `MICROPY_TASK_STACK_SIZE` to match other board-configurable variables, and probably move the default definition to mpconfigport.h.

**Comment by dotnfc** [comment]
thanks for you suggestion, i've pulled a simple request [here](https://github.com/micropython/micropython/pull/12415)
